lib/Alien/Build/Util.pm

```diff
@@ -76,7 +76,26 @@ sub _mirror
         { unlink "$dst" }
         my $target = readlink "$src";
         Alien::Build->log("ln -s $target $dst") if $opt->{verbose};
+        if (path($target)->is_relative) {
+          my $nativesymlink =
+                   (($^O eq "msys" && defined $ENV{MSYS} && $ENV{MSYS} eq "winsymlinks:nativestrict")
+                || ($^O eq "cygwin" && defined $ENV{CYGWIN} && $ENV{CYGWIN} eq "winsymlinks:nativestrict"));
+          if (!$src->parent->child($target)->exists) {
+            if ($nativesymlink) {
+              # NOTE: On linux, it is OK to create broken symlinks, but it is not allowed on 
+              #   windows MSYS2/Cygwin when nativestrict is used.
+              die "cannot create native symlink to nonexistent file $target on $^O";
+            }
+          }
+          if ($nativesymlink) {
+            $dst->parent->child($target)->touchpath;
```

Comment:
I guess this is assuming that the target will later be updated as part of the copy?  There is a corner case here though when the link really should be broken at the destination because of relative links, will leave an empty file instead.

Reply:
>  will leave an empty file instead.

That should not happen since we do the check of existence relative to the source at line 83. So if the target does not exist relative to the source, the code dies at line 87.